### PR TITLE
SDPA-3557 added queue_mail

### PR DESCRIPTION
**Pull request by vincent-gao**
### Jira: 
https://digital-engagement.atlassian.net/browse/SDPA-3557

### changes:
1. Adding queue_mail module as tide_workflow_notification's dependency
2. Adding a hook_update and configs yaml file.

relates to https://github.com/dpc-sdp/content-vic-gov-au/pull/758

### CODE REVIEW

File: modules/tide_workflow_notification/tide_workflow_notification.install [hunk 1]
```diff
@@ -15,3 +15,13 @@ function tide_workflow_notification_install() {
     user_role_grant_permissions($role_name, $permissions);
   }
 }
+
+/**
+ * Implements hook_update_N().
+ */
+function tide_workflow_notification_update_8001() {
```
Comment (anthony-malkoun) [inline modules/tide_workflow_notification/tide_workflow_notification.install]: Shouldn't you enable the module here too?

Comment (vincent-gao) [reply modules/tide_workflow_notification/tide_workflow_notification.install]: ah. yes. Thanks @anthony-malkoun

Comment (GROwen) [reply modules/tide_workflow_notification/tide_workflow_notification.install]: Is `tide_workflow_notification` installed anywhere yet? Could this just go in the install hook?

Comment (vincent-gao) [reply modules/tide_workflow_notification/tide_workflow_notification.install]: yes. @GROwen  thanks.

**Pull request merged**